src/components/Login.tsx

```diff
@@ -44,18 +49,28 @@ const inputStyle = css`
   line-height: 1.375rem;
   padding: 8px 0px;
   border-width: 0px 0px 1px;
-  border-color: ${theme.colors.textDisabled};
+  border-color: ${focused
+    ? theme.colors.gray700
+    : hasError && touched
+      ? theme.colors.red700
+      : theme.colors.textDisabled};
 
   &:focus {
-    border-color: ${theme.colors.gray700};
     outline: none;
   }
+
   &::placeholder {
     color: ${theme.colors.textPlaceholder};
-    opacity: 1; /
+    opacity: 1;
   }
 `;
 
+const errorMessageStyle = css`
+  color: ${theme.colors.red700 || 'red'};
```

Comment:
토큰은 잘 작성해주셨으니 'red'는 따로 안 적어도 되겠네요!

```suggestion
  color: ${theme.colors.red700};
```